code-challenges.test.js

```diff
@@ -18,18 +18,47 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+// var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+describe("shuffleRemainingItems", () => {
+    it("return array with first item removed and the remaining items shuffled", () => {
+        const colors1 = ["purple", "blue", "green", "yellow", "pink"];
+        const colors1Expected = ["yellow", "blue", "pink", "green"];
+        const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"];
+        const colors2Expected = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"];
 
-
+        expect( shuffleRemainingItems(colors1).sort() ).toEqual( colors1Expected.sort() );
```

Comment:
awesome job referencing different arrays and using logic to test for randomization. However, also consider that if you remove the logic in your array to shuffle elements, this test would still pass